lib/slack-ruby-bot/rspec/support/slack-ruby-bot/respond_with_slack_messages.rb

```diff
@@ -0,0 +1,30 @@
+require 'rspec/expectations'
+RSpec::Matchers.define :respond_with_slack_messages do |expected|
+  match do |actual|
+    raise ArgumentError, 'respond_with_slack_messages expects an array of ordered responses' unless expected.is_a? Array
+    client = if respond_to?(:client)
+               send(:client)
+             else
+               SlackRubyBot::Client.new
+             end
+
+    message_command = SlackRubyBot::Hooks::Message.new
+    channel, user, message = parse(actual)
+
+    allow(Giphy).to receive(:random) if defined?(Giphy)
+
+    allow(client).to receive(:message)
+    message_command.call(client, Hashie::Mash.new(text: message, channel: channel, user: user))
+    expected.each do |exp|
+      expect(client).to have_received(:message).with(channel: channel, text: exp).once
+    end
+    true
```

Comment:
Same as above, I think we want exact expectations, making sure only the messages expected have been received. This behavior should have tests in both cases.